eris.go

```diff
@@ -140,13 +140,17 @@ func Cause(err error) error {
 
 // StackFrames returns the trace of an error in the form of a program counter slice.
 // Use this method if you want to pass the eris stack trace to some other error tracing library.
+// Currently this method is for testing rootError and wrapError StackFrames() method only.
```

Comment:
Is this comment necessary? Seems like it could still be useful for some users even apart from Sentry

Reply:
I wanted to denote that it is not part of the core library but still usable